interface/main/finder/patient_select.php

```diff
@@ -158,6 +158,10 @@ function submitList(offset) {
             } else if ($field_id == 'pubpid') {
                 $where .= " AND $field_id LIKE ?";
                 array_push($sqlBindArray, $value);
+                //for 'date' field
+            } else if ($data_type == 4) {
+                $where .= " AND $field_id LIKE ?";
+                array_push($sqlBindArray, DateToYYYYMMDD($value));
```

Comment:
TODO:
analyze to see if this needed (and if so, do we need to incorporate if time option is set)

Reply:
DONE (this looks reasonable)